pkg/resourceinterpreter/default/native/aggregatestatus.go

```diff
@@ -80,14 +81,23 @@ func aggregateDeploymentStatus(object *unstructured.Unstructured, aggregatedStat
 		// which is the generation Karmada 'observed'.
 		// The 'observedGeneration' is mainly used by GitOps tools(like 'Argo CD') to assess the health status.
 		// For more details, please refer to https://argo-cd.readthedocs.io/en/stable/operator-manual/health/.
-		newStatus.ObservedGeneration = deploy.Generation
```

Comment:
This comment may no longer be appropriate:
```
// always set 'observedGeneration' with current generation(.metadata.generation)
// which is the generation Karmada 'observed'.
```

Reply:
fixed